File: programmers/js/basic/181898.js
// 가까운 1 찾기
function solution(arr, idx) {
  for (let i = 0; i < arr.length; i++) {
    if (i >= idx && arr[i] === 1) {
      return i;
    }
  }

  return -1;
}

// 다른 풀이
function solution(arr, idx) {
  return arr.indexOf(1, idx);
}

File: programmers/java/basic/181898.java
// 가까운 1 찾기

import java.util.Arrays;
import java.util.stream.Collectors;

class Solution {
    public int solution(int[] arr, int idx) {
        return Arrays.stream(arr)
            .mapToObj(String::valueOf)
            .collect(Collectors.joining(""))
            .indexOf("1", idx);
    }
}